tests/manager/manager-ctor-ssl-deprecated-002.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): Test deprecated options
```

Comment:
I'd suggest adding "(capath context driver option)" or at least "(capath)" to the description, to differentiate it from the previous test's name.